notebooks/03_spatial_data_processing.md

```diff
@@ -333,34 +276,705 @@ plt.show()
 
 This map gives us a very different view. Very large countries are all of a sudden "penalized" and some smaller ones rise to the highest values. Again, how to read this map and whether its message is interesting or not depends on what we are after. But, together with the previous one, it does highlight important issues to consider when exploring uneven spatial data and what it means to display data (e.g. airports) through specific geographies (e.g. countries).
 
---- 
 
-## Vignette: House Prices
+
+## Vignette: Airbnb Prices in San Diego
+
+Our second vignette changes the spatial scale to examine the distribution of
+Airbnb listings in San Diego California. We begin by reading the listings, 
+ which are stored in a comma separated file (csv):
+
 
 ```python
 df = pd.read_csv('../data/sandiego/listings.csv')
 len(df)
+
+```
+
+Examination of the dataframe reveals that the location information is encoded
+in the columns `longitude` and `latitude`.  We will use these columns together
+with the `Point` class from **shapely** to create a geodataframe:
+
+```python
+from shapely.geometry import Point
+```
+
+```python jupyter={"outputs_hidden": false}
+geometry = [Point(xy) for xy in zip(df['longitude'], df['latitude'])]
+```
+
+```python
+import geopandas as gpd
+```
+
+```python
+gdf = gpd.GeoDataFrame(df)
+```
+
+```python
+gdf['geometry'] = geometry
+```
+
+```python
+gdf.plot()
+```
+
+The listings are represented as point objects, and what we are interested in is
+the spatial distribution of the listing prices across the market. So unlike the
+airport dataset above where we were focusing on the locations of the points,
+here we extend the analysis to consider the locations and an attribute that is
+measured at each of the locations.
+
+Before we analyze the spatial distribution of the listings, we have to convert
+the data type of the listing column as it is currently encoded as a string:
+
+```python
+gdf.price
+```
+
+```python
+gdf['price'] = gdf['price'].apply(lambda x: x.replace('$', '').replace(',', '')).astype(float)
```

Comment:
This should be broken up into parts and explained. `.str` is pretty unique.

Reply:
Will resolve in my branch